proposed/cache.md

```diff
@@ -255,16 +255,6 @@ interface CacheItemInterface
      */
     public function expiresAfter($time);
 
```

Comment:
I don't recall if extra blank lines in interfaces is against PSR-2 or not, but I wanted to mention it in case someone else thinks it might. I also understand this is inline code so not sure how picky we need to be following PSR-2 for this.